tasks/system/Linux.yml

```diff
@@ -69,15 +70,6 @@
       arch: "{{ 'x86-32' if ansible_architecture == 'i386' else 'x86-64' }}"
     when: sonar_installation_result.files is defined
```

Comment:
Does it make sense to check that sonar_installation_result is defined?

Reply:
No, it's leftover from removed task.